packages/components/src/components/tag/_tag.scss

```diff
@@ -81,14 +84,26 @@
     }
   }
 
+  .#{$prefix}--tag__label {
+    max-width: 100%;
+    overflow: hidden;
+    white-space: nowrap;
+    text-overflow: ellipsis;
+  }
+
   // tags used for filtering
   .#{$prefix}--tag--filter {
     @include tag-theme($inverse-02, $inverse-01);
+
     cursor: pointer;
-    padding-right: rem(2px); // Align with hover circle of X button
+    padding-right: rem(26px); // Space for hover circle and X button
```

Comment:
just to confirm the 26px is from 20px width svg, 2px left border, 2px right border, and 2px space from the right edge right? should we express it as

`padding-right: calc(#{$carbon--spacing-06} + #{rem(2px)}); // icon width + 2px space from right edge` ?

Reply:
bump @lee-chase if you have a sec!